### PR TITLE
[TinkerForge Action] Correct linter errors

### PR DESCRIPTION
**Pull request by bgilmer77**
Signed-off-by: Brad Gilmer <brad@gilmer.tv>

### CODE REVIEW

File: bundles/action/org.openhab.action.tinkerforge/README.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ These action functions are available:
...
+*   `tfServoSetposition(String uid, String num, String position, String velocity, String acceleration)`
 
-  Sets the position of a TinkerForge servo with the uid $uid and servo number to the position $position using the speed $speed and acceleration $acceleration.
+Sets the position of a TinkerForge servo with the uid $uid and servo number to the position $position using the speed $speed and acceleration $acceleration.
 
-  Example:
+Example:
 
-	```
+	```java
```
Comment (9037568) [inline bundles/action/org.openhab.action.tinkerforge/README.md]: This appears to be broken. "```java" appears in the block when viewing the file, whereas this does not occur in the "clear lcd" example above.

File: bundles/action/org.openhab.action.tinkerforge/README.md [hunk 2]
```diff
@@ -44,16 +44,16 @@ These action functions are available:
...
-	* speed: value between -32767 - 32767
-	* drivemode is either "break" or "coast"
+*   speed: value between -32767 - 32767
+*   drivemode is either "break" or "coast"
 
-	Example:
+Example:
 
-	```
+	```java
```
Comment (9037568) [inline bundles/action/org.openhab.action.tinkerforge/README.md]: Same problem as above.

File: bundles/action/org.openhab.action.tinkerforge/README.md [hunk 3]
```diff
@@ -70,16 +70,16 @@ These action functions are available:
...
-  * speed: value between -32767 - 32767
-  * drivemode is either "break" or "coast"
+*   speed: value between -32767 - 32767
+*   drivemode is either "break" or "coast"
 
-  Example:
+Example:
 
-	```
+	```java
```
Comment (9037568) [inline bundles/action/org.openhab.action.tinkerforge/README.md]: Same problem as above.

File: bundles/action/org.openhab.action.tinkerforge/README.md [hunk 4]
```diff
@@ -102,25 +102,26 @@ These action functions are available:
 
   Example:
 
-  ```
+  ```java
```
Comment (9037568) [inline bundles/action/org.openhab.action.tinkerforge/README.md]: Same problem as above.

**Comment by bgilmer77** [comment]
I have been talking about this with @Confectrician and @ThomDietrich.  It seems that the javascript code fence works better in this case.  We are not exactly sure why.  Please let me know if this looks okay and merge if it does.

**Comment by ThomDietrich** [comment]
@bgilmer77 The current result doesn't look correct at all. There seem to be some indention issues, also you did not replace asterisks as the list marker. Are you sure you are using out linter settings?

**Comment by bgilmer77** [comment]
Hi @ThomDietrich out of time today.  I will look carefully at this tomorrow.

**Comment by 9037568** [comment]
Thanks, @bgilmer77 !

**Pull request merged**